connectors/citrus-openapi/src/main/java/org/citrusframework/openapi/OpenApiTestDataGenerator.java

```diff
@@ -32,6 +32,7 @@
  * enforce the specification rules.
  *
  * @author Christoph Deppisch
+ * @author Ralf Ueberfuhr
```

Comment:
```suggestion
```

sorry to bother you again, but please remove these. see https://github.com/citrusframework/citrus/pull/1171#discussion_r1619210698.

Reply:
Okay, if this is disturbing, I'll remove the @author tag (but only my own) to minimize the diffs.